.github/workflows/release.yml

```diff
@@ -44,6 +44,8 @@ jobs:
     - uses: actions/setup-node@v3
       with:
         node-version: 16
+    - name: Run npm audit fix 
+      run: npm audit fix
```

Comment:
Without pushing back to github is this a good idea? You could end up deploying a version that just not been tested and have a breaking change that slips under the radar

Reply:
Yeah, we just had a similar debate and then closed the PR. Thanks for confirming Jordan